WebClock/ClockInOutRepository.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using WebClock.Models;
+
+namespace WebClock
+{
+    public class ClockInOutRepository
```

Comment:
Jeśli przyjmujemy, że jednak jest to jakaś forma repozytorium, to ta klasa powinna być raczej umieszczona w namespace WebClock.Models.